File: Java Solutions/RT_TRAINGLE.java
/*
 * Author: Rajat Gupta
 */

import java.io.*;

class Solution {
    public static void main(String[] args) {
        
        String num = null;
        int n, i, j;

        try {
            BufferedReader is = new BufferedReader(
                    new InputStreamReader(System.in));
            num = is.readLine();
        } 
        catch( IOException e) {
            System.out.println("IOException: " + e);
        }

        n = Integer.parseInt(num);

        for(i=1; i<=n; i++) {
            for(j=1; j<=i; j++) {
                System.out.print("*");
            }
            System.out.print("\n");
        }

    }
}

File: RT_TRAINGLE.c
/*
 * Author: Rajat Gupta
 */

#include <stdio.h>

#define read(n) scanf("%d", &n)

int main() {
    int n, i, j;
    read(n);
    for(i=1; i<=n; i++) {
        for(j=1; j<=i; j++) {
            printf("*");
        }
        printf("\n");
    }
    return 0;
}
